### PR TITLE
Run all unit tests (Fixed Lint)

### PR DESCRIPTION
**Pull request by davebcn87**
# Short description 📝

There were some changes to fourier and seems like we have been running for a few days only swift run tuist test -> this means only tests defined via tuist itself would be run. To fix this, we should additionally run swift test as well.

I have also fixed a bunch of failing tests that crept into the codebase because of this.

This is also blocking the current release, unfortunately.

# Checklist ✅

  The code architecture and patterns are consistent with the rest of the codebase.
  The changes have been tested following the documented guidelines.
  The CHANGELOG.md has been updated to reflect the changes. In case of a breaking change, it's been flagged as such.
  In case the PR introduces changes that affect users, the documentation has been updated.

### CODE REVIEW

**Comment by davebcn87** [comment]
@fortmarek here is your PR with `./fourier lint all` corrected

**Review by fortmarek — APPROVED** [review]
Thanks! Let's merge this when CI becomes green

File: projects/fourier/lib/fourier/services/test/tuist/unit.rb [hunk 1]
```diff
@@ -6,6 +6,9 @@ module Tuist
         class Unit < Base
           def call
             Utilities::System.tuist("test")
+            Dir.chdir(Constants::TUIST_DIRECTORY) do
+              Utilities::System.system("swift", "test")
```
Comment (fortmarek) [inline projects/fourier/lib/fourier/services/test/tuist/unit.rb:10]: @pepibumur could you afterwards look at this if it makes sense? I was not completely sure about this one. But `swift run tuist test` was not running all the tests.

Comment (pepibumur) [reply projects/fourier/lib/fourier/services/test/tuist/unit.rb:10]: Right. This is correct.

**Pull request merged**